modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableHeaderRow.java

```diff
@@ -627,25 +634,29 @@ private void createMenuItem(TableColumnBase<?,?> col) {
         if (item == null) {
             item = new CheckMenuItem();
             columnMenuItems.put(col, item);
+
+            item.setSelected(col.isVisible());
+
+            final CheckMenuItem _item = item;
+            // fake bidrectional binding (a real one was used here but resulted in JBS-8136468)
```

Comment:
that's JDK-8136468

Reply:
maybe we we should do a small cleanup ticket where we replace all `JBS-` and `RT-` references with the `JDK-` one? A quick check led to around 120 occurrences.

Reply:
and maybe fix the RT- references as well, though it might be a bit harder because a  lookup is required.

Reply:
That is correct. I still think it's worth it, especially since not everyone knows the 'trick' of pasting the `RT-` entry into the bug tracker to get the corresponding `JDK-` entry (in fact, I only learned that this year 😄)

Reply:
I'd review that!  The only question is how to do RT- efficiently, so it can be easily reviewed (without going through manually copying and pasting every id)...

Reply:
It would be pretty easy to do it with a JIRA REST query: Given an input list of `RT-nnnn` bug IDs, produce the corresponding output list of `JDK-mmmmmmm` bug IDs. I'll volunteer to do this part of it.

Someone could then easily turn that into a script using `sed` to do the replacement.

Reply:
I filed [JDK-8344899](https://bugs.openjdk.org/browse/JDK-8344899) to track the cleanup and attached a `grep.log` file of all occurrences of `"RT-[0-9]+"` and a `rt2jdk.txt` file with the mapping from `RT-nnnn` to `JDK-mmmmmmm`.